src/natcap/invest/sdr/sdr.py

```diff
@@ -90,7 +90,7 @@
             "about": (
                 "A GDAL-supported raster file, with an integer LULC code "
                 "for each cell."),
-            "name": "Land-Use/Land-Cover"
+            "name": "land use/land cover"
```

Comment:
whoops, I missed that one. I like "landcover" to be brief and consistent. Thoughts?

Reply:
Ah, I'm torn.  `landcover` is nice and short, but typically these numeric classes represent not only what's on the landscape but also how it's used.  Having intensive agriculture, for example, is a different land use, but maybe isn't a different land cover.  So I might err on the side of "land use/land cover"

But also, since LULC inputs are used in a bunch of our models, I wonder if we should (and even can) have a centralized definition of this so that if we do want to change the label, it can be done everywhere, similar to `validation.WORKSPACE_SPEC` and `validation.N_WORKERS_SPEC`.  If so, we could more easily change this across the board if a change is needed.

Reply:
Oh, that's a good point. In that case "land use/land cover" sounds good. 

> But also, since LULC inputs are used in a bunch of our models, I wonder if we should (and even can) have a centralized definition of this so that if we do want to change the label, it can be done everywhere, similar to validation.WORKSPACE_SPEC and validation.N_WORKERS_SPEC.

I really like that idea! A centralized definition for the name would definitely help. It seems our LULC inputs also always have `"type": "raster"`, and every one I've looked at has `"required": True` and `"validation_options": {"projected": True }`. The `about` property usually has a model-specific description, but maybe that's not necessary - something like "A map of land use/land cover codes where each code exists in the biophysical table" would be fine.

Were you thinking of centralizing some/all of these other properties as well?

Reply:
> Were you thinking of centralizing some/all of these other properties as well?

Ha, I started out thinking "there's not much variation in the InVEST LULCs" ... I was wrong!

I just looked through every InVEST model, and there are 7 (NDR, SDR, FCEE, AWY, Scenario Gen. Proximity, SWY, UFRM) that do have a consistent specification:

```python
LULC_RASTER_SPEC = {
    "type": "raster",
    "required": True,
    "validation_options": {
        "projected": True,
    },
    "about": "A GDAL-supported raster where each code exists in the biophysical table.",
    "name": "land use / land cover",
}
```

The remaining models vary a bit depending on the use case:

* Some extend the `about` text to provide more helpful context (Crop Regression, Crop Percentile, Pollination, UCM)
* Some use multiple LULC rasters as inputs and need more information in the labels (Carbon, HQ)
* Not all of the models that use LULCs are required (GLOBIO, HQ)
* Some are required to be projected in meters (Crop Regression, Crop Percentile, UCM)

This much variation is enough to wonder if a centralized definition is really required or helpful.

So as a result, now I'm not so sure that a centralized definition _would_ be helpful.  Sure, we could use it in the 7 that are for sure the same, but I'm not so sure that it would be easier to have to make modifications to the centralized definition in order to make it work.  But maybe there is a way to do that that makes sense?

What do you think?

And in any case, all that to say that we can safely standardize the label, and maybe a centralized LULC definition is lower priority and a separate issue.